CoastalResilienceNetwork/GeositeFramework
# Replace proxy with CORS whitelist

## Overview

In place of the built-in proxy server, use the CORS functionality built into the ESRI JS API. For most cases, this serves the same purpose as the proxy, but is more modern.

Connects to #1097

## Testing Instructions

- Use the build tool on the `static-site` branch to build a region on this branch, for example:

```
python build.py gulfmex-region --region-branch master --framework-branch cpc/proxy-fix
``` 

- Run the site.
- Activate each plugin, interact with it and verify that no proxy errors are thrown (you would see a large message in the console).
- Repeat the above process with a different region (I tried gulfmex, new jersey).
